RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -903,7 +904,11 @@ HcalNoiseInfoProducer::fillrechits(edm::Event& iEvent, const edm::EventSetup& iS
       summary.rechitCount15_ = summary.rechitCount15_ + 1;
       summary.rechitEnergy15_ = summary.rechitEnergy15_ + rechit.eraw();
     }
-
+    
+    // Exclude uncollapsed QIE11 channels
+    if(((rechit.auxPhase1()>>HBHERecHitAuxSetter::OFF_TDC_TIME)&1) &&
+       ((rechit.auxPhase1()>>HBHERecHitAuxSetter::OFF_COMBINED)&0) ) continue;
```

Comment:
isn't ```anything  & 0 ``` equal to 0 ?

Reply:
You are right. This is fixed in the new commit.